concepts/05 UI Components/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,171 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different [validation rule types](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/). 
+
+To apply validation rules to a simple item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#validationRules) array. You can specify an item's [isRequired](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property to implicitly apply the [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/) to it.
+
+In the following example, we specify the [isRequired](Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#isRequired) property for the `Name` item. We also specify the **NumericRule** for `officeNumber` and **EmailRule** for `Email` items.
+
+An editor is validated when its value changes. To validate all editors simultaneously, call the [validate()](/Documentation/ApiReference/UI_Components/dxForm/Methods/#validate) Form method. 
```

Comment:
This sentence looks out of place. Let's modify the Submit the Form topic a bit so that it includes validation before submission.